sda-commons-cacertificates/src/main/java/org/sdase/commons/cacertificates/CaCertificatesConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.cacertificates;
+
+import javax.net.ssl.SSLContext;
+import org.sdase.commons.cacertificates.ssl.CertificateReader;
+import org.sdase.commons.cacertificates.ssl.SslUtil;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+
+@AutoConfiguration
+public class CaCertificatesConfiguration {
+  private final CertificateReader certificateReader;
+
+  public CaCertificatesConfiguration(
+      @Value("${sda.caCertificates.certificatesDir:/var/trust/certificates}")
+          String defaultTrustedCertificatesDir) {
+    certificateReader = new CertificateReader(defaultTrustedCertificatesDir);
+  }
+
+  @Bean
+  @ConditionalOnMissingBean
+  public SSLContext mongoSSlContext() {
```

Comment:
Should we name it without _mongo_ in this context?
I assume, that we will use the same `SSLContext` for clients and if possible for Kafka later.

Reply:
right, renamed